### PR TITLE
add new commands to dragon.py

### PR DESCRIPTION
**Pull request by alexboche**
I am redoing previous pull request #418 because I'm having some issues with git.
I have tried to respond to the comments there.
I took out the proposed commands for ignoring what you say if you say a specific word, the reason being that I don't know what word to use since people may say that word in sentences and thus have their dictation ignored accidentally.

### CODE REVIEW

**Comment by kendonB** [comment]
new contents are just added below the old contents. See "Files changed".

**Comment by alexboche** [comment]
Sorry about that. Should be fixed now

File: castervoice/apps/dragon.py [hunk 1]
```diff
@@ -36,11 +63,12 @@ class DragonRule(MergeRule):
         'normal mode':
             R(Playback([(["normal", "mode", "on"], 0.0)]),
               rdescript="Dragon: Normal Mode"),
-        'command mode':
+        'com on':
```
Comment (kendonB) [inline castervoice/apps/dragon.py]: keep the old one as an option?

File: castervoice/apps/dragon.py [hunk 2]
```diff
@@ -36,11 +63,12 @@ class DragonRule(MergeRule):
...
             R(Playback([(["command", "mode", "on"], 0.0)]),
-              rdescript="Dragon: Command Mode"),
-        'scratch':
-            R(Playback([(["scratch", "that"], 0.0)]), rdescript="Dragon: 'Scratch That'"),
+              rdescript="Dragon: Command Mode (On)"),
+        'com off':
```
Comment (kendonB) [inline castervoice/apps/dragon.py]: Good to have a verbose version of this too.

File: castervoice/apps/dragon.py [hunk 3]
```diff
@@ -55,24 +83,77 @@ class DragonRule(MergeRule):
...
+        "cory <text>": 
+            R(Mimic("correct", extra="text") + WaitWindow(title="spelling window") + Mimic("choose", "one"),
+                rdescript="Dragon: brings up the correction menu for the phrase spoken in the command and chooses the 1st choice"),
+        "cory that": 
+            R(Mimic("correct", "that") + WaitWindow(title="spelling window") + Mimic("choose", "one"), 
+                rdescript="Dragon: brings up the correction menu for the previously spoken phrase and chooses the first choice"),
+
+        "make that <text>": R(Mimic("scratch", "that") + Mimic(extra="text"), 
+             rdescript="Dragon: deletes the dictation generated by the previous utterance and replaces it with what you say next"),
+        'scratch [<n10>]': R(Playback([(["scratch", "that"], 0.03)]),
```
Comment (kendonB) [inline castervoice/apps/dragon.py]: double quotes for consistency

**Comment by kendonB** [comment]
Still needs testing (by me) but looks pretty good so far. We will add docs right before merging as well.

**Comment by alexboche** [comment]
So this pull request is done and tested by me. (maintainers may want to tested). I would remove the "waiting for user reply" tag unless I'm missing something.

**Pull request merged**